src/collective/solr/utils.py

```diff
@@ -4,23 +4,30 @@
 
 from Acquisition import aq_base
 from unidecode import unidecode
-from zope.component import queryUtility
 
-from collective.solr.interfaces import ISolrConnectionConfig
+from plone import api
+from collective.solr.interfaces import ISolrSchema
+from zope.component import getUtility
+from plone.registry.interfaces import IRegistry
+
+
+def getConfig():
```

Comment:
@tomgross @do3cc the getConfig method is called during the unit tests. the lookup always fails because there is no Plone fixture present at all.